ui/src/components/nested-fields/QasNestedFields.vue

```diff
@@ -381,6 +388,24 @@ export default {
       return list
     },
 
+    getFields (index, row) {
+      const fields = this.children
+
+      if (typeof this.fieldsHandlerFn === 'function') {
```

Comment:
fieldsHandlerFn ou é função ou undefined certo? ao invés de toda linha ele validar o typeof, não poderia ser uma computada `hasFieldsHandlerFn`? assim ele faz o cache...

Reply:
ou somente `if(this.fieldsHandlerFn)` já não resolve?

Reply:
Feito